Assignment4/andreapisa9-CA67270LK/task09.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""Task09.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1GhwcX1Icuj9Vfpy16dWOre48OKYcaPMk
+
+**Task 09: Data linking**
+"""
+
+#!pip install rdflib
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials/"
+
+from rdflib import Graph, Namespace, Literal, URIRef
+g1 = Graph()
+g2 = Graph()
+g3 = Graph()
+g1.parse(github_storage+"rdf/data03.rdf", format="xml")
+g2.parse(github_storage+"rdf/data04.rdf", format="xml")
+
+"""Busca individuos en los dos grafos y enlázalos mediante la propiedad OWL:sameAs, inserta estas coincidencias en g3. Consideramos dos individuos iguales si tienen el mismo apodo y nombre de familia. Ten en cuenta que las URI no tienen por qué ser iguales para un mismo individuo en los dos grafos."""
+
+from rdflib import XSD
+from rdflib.namespace import RDF, RDFS, OWL
+from rdflib.plugins.sparql import prepareQuery
+
+ns = Namespace("http://data.three.org#")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+#Find identities in the 2 graphs:
+#1. Get all instances of Person in Graph 1
+q1 = prepareQuery('''
+                  SELECT ?Subject WHERE{
+                    ?Subject a ns:Person
```

Comment:
The graphs have different namespaces for class Person (for some reason), and therefore this would yield the persons only in 1 graph, not both